ocs_ci/utility/pagerduty.py

```diff
@@ -247,15 +287,21 @@ def get_service_dict(self):
         cluster_name = config.ENV_DATA["cluster_name"]
         # timestamp is added to service name to ensure unique name of service
         timestamp = time.time()
-        default_policy = self.get_default_escalation_policy_id()
+        policy = None
+        policy_name = config.AUTH["pagerduty"].get("escalation_policy")
+        if policy_name:
+            policy = self.get_escalation_policy_id(policy_name)
+            if not policy:
+                raise ValueError(f"Policy {policy_name} not found")
+        policy = policy
```

Comment:
`policy = policy` line doesn't really make sense to me, could you please explain?

Reply:
You are correct. I will delete it.